templates/artist/artist_detail.html

```diff
@@ -211,8 +216,8 @@ <h1 class="header-font">{{ artist.name }}</h1>
                 {% for update in updates %}
                     <li>{{ update.created_datetime }}: {{ update.text }}</li>
                 {% endfor %}
-            </li>
             </ul>
```

Comment:
Awesome. Now we should indent lines 214-219 so that this section is inside the `<li>`.